src/main/java/com/google/collegeplanner/servlets/DepartmentServlet.java

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.collegeplanner.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.http.client.utils.URIBuilder;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+/** Servlet that returns list of departments.*/
+@WebServlet("/api/departments")
+public class DepartmentServlet extends HttpServlet {
+  ApiUtil apiUtil;
+
+  public DepartmentServlet() {
+    this.apiUtil = new ApiUtil();
```

Comment:
try `this(new ApiUtil());` to consolidate a tiny bit

https://stackoverflow.com/questions/285177/how-do-i-call-one-constructor-from-another-in-java